src/parser/monk/mistweaver/modules/spells/InvokeYulon.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+
+import Analyzer, { SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+
+import SpellIcon from 'common/SpellIcon';
+import StatisticBox, { STATISTIC_ORDER } from 'interface/others/StatisticBox';
+
+import { formatNumber } from 'common/format';
+import { TooltipElement } from 'common/Tooltip';
+import ItemHealingDone from 'interface/ItemHealingDone';
+
+class InvokeYulon extends Analyzer {
+  petID = null;
+  soothHealing = 0;
+  envelopHealing = 0;
+
+  constructor(...args) {
+    super(...args);
+    this.active = !this.selectedCombatant.hasTalent(SPELLS.INVOKE_CHIJI_THE_RED_CRANE_TALENT.id);
```

Comment:
Right after this you should add 
`if(!this.active){
return;
}
`
To prevent adding unneeded action listeners